debezium-embedded/src/test/java/io/debezium/embedded/AbstractConnectorTest.java

```diff
@@ -63,7 +63,6 @@
 import org.awaitility.core.ConditionTimeoutException;
```

Comment:
There should be more files in the `rest` package that were only used by RestExtension classes and the schema generator and can be removed too.
I think the schema generator module can be removed too. We can revisit the git history when we need to bring parts of it back for the Debezium Platform eventually. wdyt @jpechane @mfvitale ?